Makefile

```diff
@@ -1,115 +1,98 @@
-.PHONY: uninstall install setup-cli get-master-ip launch-dcos detroy-dcos docker-build docker kubectl-tunnel deploy
+.PHONY: azure aws gce uninstall install get-cli setup-cli get-master-ip get-master-elb_ip plan-dcos launch-dcos detroy-dcos kubectl-config kubectl-tunnel plan deploy
 
 RM := rm -f
-LAUNCH_CONFIG_FILE := launch.yaml
-CLUSTER_INFO_FILE := cluster_info.json
-BUILD_FILE := .output
-MASTER_IP_FILE := .master_ip
-ID_FILE := .id_key
 SSH_USER := core
-DCOS_INSTALLER_URL := https://downloads.dcos.io/dcos/stable/1.10.1/dcos_generate_config.sh
-
-include ./resources/gce.template.mk
-include ./resources/aws.template.mk
-include ./resources/azure.template.mk
+MASTER_IP_FILE := .master_ip
+MASTER_ELB_IP_FILE := .master_elb_ip
+TERRAFORM_INSTALLER_URL := github.com/dcos/terraform-dcos
+DCOS_LAUNCH_VERSION := 0.5.7
+KUBERNETES_VERSION := v1.7.10
+
+get-cli:
+	$(eval export DCOS_LAUNCH_VERSION)
+	$(eval export KUBERNETES_VERSION)
+	scripts/get_cli
+
+azure: clean
+	mkdir .deploy
+	cd .deploy; \
+	cp ../resources/desired_cluster_profile.azure desired_cluster_profile; \
+	terraform init -from-module $(TERRAFORM_INSTALLER_URL)//azure
+
+aws: clean
+	mkdir .deploy
+	cd .deploy; \
+	cp ../resources/desired_cluster_profile.aws desired_cluster_profile; \
+	terraform init -from-module $(TERRAFORM_INSTALLER_URL)//aws
+
+gce: clean
+	$(RM) -r .deploy
+	mkdir .deploy
+	cd .deploy; \
+	cp ../resources/desired_cluster_profile.gce desired_cluster_profile; \
+	terraform init -from-module $(TERRAFORM_INSTALLER_URL)//gcp; \
+	rm desired_cluster_profile.tfvars.example
```

Comment:
Why is this here and not on `aws` and `azure`?

Reply:
only gcp `desired_cluster_profile.tfvars.example` has this file

Reply:
But why do we need to remove it?